mflowgen/common/rtl/gen_rtl.sh

```diff
@@ -103,8 +165,9 @@ else
       # make to generate systemRDL RTL files for global controller
       make -C $GARNET_HOME/global_controller rtl CGRA_WIDTH=${array_width}
       cat global_controller/systemRDL/output/*.sv >> $current_dir/outputs/design.v
-    fi
 
-    cd $current_dir
+      cd $current_dir ; # why? all we do after this is exit back to calling dir...?
```

Comment:
1) I moved this `cd` bask inside the `else` block where it obviously belongs although 2) the only thing that happens after this is that the script exits, so the `cd` is useless AFAICT but 3) I wasn't brave enough to remove it entirely.